.travis.yml

```diff
@@ -1,3 +1,6 @@
+arch:
+  - amd64
+  - ppc64le
 language: python
 python:
     - "2.7"
```

Comment:
It would make sense to test this also with newer python3, not only 3.4, but better 3.8 or 3.9

Reply:
Hi @norbusan  Thanks for suggetion, I have added one of the latest python 3.7 and 3.8. Please have a look on this https://travis-ci.com/github/sanjaymsh/debian-speg/builds/209613718.

Thanks !!